autosubmit/job/job_list.py

```diff
@@ -1377,14 +1369,16 @@ def _manage_job_dependencies(self, dic_jobs, job, date_list, member_list, chunk_
             if dic_jobs.as_conf.jobs_data.get(aux_key, {}).get("RUNNING", "once") == "chunk":
                 if aux_key in distances_of_current_section and distance > distances_of_current_section[aux_key]:
                     distances_of_current_section[aux_key] = distance
+
             elif dic_jobs.as_conf.jobs_data.get(aux_key, {}).get("RUNNING", "once") == "member":
-                if aux_key in distances_of_current_section_member and distance > distances_of_current_section_member[aux_key]:
+                if distances_of_current_section_member.get(aux_key, False) < distance:
```

Comment:
Ah, my bad! This would introduce a bug. I hadn't seen the `< distance` yesterday (risk of reviewing code in the afternoon without :coffee: , sorry!). We will have to keep the code as was. If you want to fix the column length, then this should do.

```py
                if (aux_key in distances_of_current_section_member and 
                        distance > distances_of_current_section_member[aux_key]):
```

Or breaking it at some other point.